docs/learn/learn-proxies.md

```diff
@@ -141,20 +157,27 @@ next to it). The `addProxy(proxy, proxy_type)` function will need to be selected
 a proxy. The chosen proxy account that you set will be the account that has the proxy on it. The
 selected account at the top is the account that will be the primary account.
 
-> NOTE: If you see an `unused` option when adding in a proxy, this is not a proxy type. This is an
-> empty `enum`, and if you try to add this in as a proxy, nothing will happen. No new proxy will be
-> created.
+:::caution `unused` option
+
+If you see an `unused` option when adding in a proxy, this is not a proxy type. This is an
+empty `enum`, and if you try to add this in as a proxy, nothing will happen. No new proxy will be
+created.
+
+:::
 
 **It is critical to setup Anonymous Proxies with appropriate permissions and be aware of potential dangers**
 
 ### Creating Anonymous Proxies on Polkadot-JS UI
 
-> NOTE: The first anonymous proxy you add should always be of *type* `Any`.
-> Also, if there are multiple anonymous proxies for the proxied account, you should keep at least  *one* `Any`
-> type.
-> Without having an `Any` type proxy, you won't be able to send funds, add new proxies, kill the
-> anonymous proxy or take any action not specifically allowed by the types of the proxies the
-> account has.
+:::caution Anonymous proxy types
```

Comment:
This caution is about setting at least one anonymous proxy of **any** type. Anonymous proxy types title is misleading. The caution should be okay without a title
```suggestion
:::caution 
```